src/backend/sqlScripts/3_user_and_permissions_tables.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE Users (
+	id SERIAL PRIMARY KEY,
+	username varchar(25),
+	preferred_name varchar(50),
+	passhash varchar(512),
+	email varchar(100),
+	permission_id int,
+    FOREIGN KEY (permission_id) REFERENCES Permissions(id)
+);
+
+CREATE TABLE Permissions (
```

Comment:
As industry standard, we tend to call these _roles_.

So each _role_ has certain _permissions_ to do things. (Permissions wouldn't exist as objects or tables like this one here - they'd just be whatever authorisation measures we put into place, like not allowed users to insert quotations into the quotations table.)

Remember to update all instances of "permission", including in the filename!